packages/typescript-eslint/tests/config-helper.test.ts

```diff
@@ -247,4 +245,16 @@ describe('config helper', () => {
       { rules: { rule: 'error' } },
     ]);
   });
+
+  it.each([undefined, null])(
```

Comment:
(testing) maybe add also like one string and/or number or something 🤷‍♂️ 

```suggestion
  it.each([undefined, null, "invalid config object", 42])(
```